tests/kola/rpm-ostree/replace-rt-kernel

```diff
@@ -2,6 +2,10 @@
 ## kola:
 ##   tags: "needs-internet"
 ##   timeoutMin: 30
+##   # We've seen some OOM when 1024M is used in similar tests:
+##   # https://github.com/coreos/fedora-coreos-tracker/issues/1506
+##   minMemory: 2048
+##   distros: rhcos
```

Comment:
We don't have an scos switch here, `rhcos` is aliased to `scos`. You'll have to keep the denylist entry above but just for SCOS so `c9s`.

Reply:
It would be nice if we could drop the denylist entry.. Maybe we use [`is_scos()`](https://github.com/coreos/fedora-coreos-config/blob/8807c8d9b3be1a6958bdb7016c70e5bdcf15bec6/tests/kola/data/commonlib.sh#L62-L66) inside the test?

If the reason it's failing is because of https://github.com/openshift/os/pull/1309#issuecomment-1587660800 then we could just do the rt test on SCOS, which is still a pretty good test.